CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - Add new `RSpec/FactoryBot/FactoryNameStyle` cop. ([@ydah])
 - Fix wrong autocorrection in `n_times` style on `RSpec/FactoryBot/CreateList`. ([@r7kamura])
+- Fix a false positive for `RSpec/FactoryBot/ConsistentParenthesesStyle` when using `generate` with not a one argument. ([@ydah])
```

Comment:
```suggestion
- Fix a false positive for `RSpec/FactoryBot/ConsistentParenthesesStyle` when using `generate` with multiple arguments. ([@ydah])
```

Reply:
It's "zero or more than one argument"

Reply:
The code changes is only about multiple arguments. Perhaps 0 were already handled by the matcher. We even have an existing test `it 'ignores FactoryBot DSL methods without a first positional argument' do`

Reply:
Thank you so much. I updated it.